pkg/codegen/go/gen.go

```diff
@@ -782,6 +782,7 @@ func (pkg *pkgContext) genEnumType(w io.Writer, name string, enumType *schema.En
 	case "number":
 		goElementType = "float64"
 	}
+	asFuncName := Title(strings.Replace(elementType, "pulumi.", "", -1))
```

Comment:
```suggestion
	asFuncName := strings.TrimPrefix(elementType, "pulumi.")
```

I don't think we need the call to `Title` here, do we? If we expect `elementType` to always be `pulumi.Foo`, then the result of trimming the prefix will always be exported. Or are there cases where we don't expect to see that prefix?

Reply:
You're probably right! This was just moving around some existing code.